src/style.css

```diff
@@ -1,3 +1,85 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+li,
+a {
+  text-decoration: none;
+}
```

Comment:
It's not recommended to use tag names for styling. It's better to style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.